mission_control/navigator_missions/navigator_singleton/pose_editor.py

```diff
@@ -216,16 +230,41 @@ def to_lat_long(self, lat, lon, alt=0):
         enu_vector[2] = 0  # We don't want to move in the z at all
         return self.set_position(enu_pos + enu_vector)
 
-    def circle_point(self, point, radius, granularity=8, theta_offset=0):
-        '''
-        Circle a point whilst looking at it
+    def spiral_point(self, point, direction='ccw', revolutions=1, meters_per_rev=0):
+        """
+        Sprials an ENU point.
+
+        `point` is a 2d or 3d numpy point or geometery_msgs/PointStamped to spiral
+        `direction` can be 'cw' or 'ccw' for clockwise or counter clockwise
+        `meters_per_rev` is the number of meters per revolution to increase the radius of the spiral
+        `revolutions` is the number of revolutions to complete
+
+        NOTE: Don't use this function with other pose editor functions like you traditionally would
+        """
```

Comment:
Yeah, don't do `navigator.move.yaw_left(...).circle_point(...).forward(...).go()`
Circling and spiraling is handled differently at the planner level and so it should be treated differently here too.